src/World.h

```diff
@@ -855,16 +855,6 @@ class cWorld  // tolua_export
 	Returns true if grown, false if not (invalid chunk, non-growable block, already ripe). */
 	bool GrowRipePlant(Vector3i a_BlockPos);
 
-	/** OBSOLETE, use the Vector3-based overload instead.
-	Grows the plant at the specified block to its ripe stage.
-	a_IsByBonemeal is obsolete, do not use.
-	Returns true if grown, false if not (invalid chunk, non-growable block, already ripe). */
-	bool GrowRipePlant(int a_BlockX, int a_BlockY, int a_BlockZ, bool a_IsByBonemeal = false)
-	{
-		UNUSED(a_IsByBonemeal);
-		return GrowRipePlant({a_BlockX, a_BlockY, a_BlockZ});
-	}
-
 	/** Grows a cactus present at the block specified by the amount of blocks specified, up to the max height specified in the config; returns the amount of blocks the cactus grew inside this call */
```

Comment:
If we remove this we might want to add this to the deprecated bindings

Reply:
It's been tagged as obsolete in the API docs [here](https://api.cuberite.org/cWorld.html#GrowRipePlant_2) but I agree a warning wouldn't hurt.

Reply:
I ended up putting a warning directly in the header file, since I'd have to deal with the Vector3i variant as well if I exported manually with DeprecatedBindings.cpp. Also thought it's about time to remove the GrowCactus/GrowMelonPumpkin ones which did print warnings.